keras_cv/layers/preprocessing/aug_mix.py

```diff
@@ -0,0 +1,295 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv import layers
+from keras_cv.utils import preprocessing
+
+
+@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class AugMix(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
+    """Performs the AugMix data augmentation technique.
+
+    Args:
+        value_range: the range of values the incoming images will have.
+            Represented as a two number tuple written [low, high].
+            This is typically either `[0, 1]` or `[0, 255]` depending
+            on how your preprocessing pipeline is setup.
+        severity: an integer representing the level of strength of
+            augmentations (between 1 and 10). Defaults to 3.
+        width: an integer representing the number of different chains to
+            be mixed. Defaults to 3.
+        depth: an integer representing the number of transformations in
+            the chains. A negative value enables stochastic depth uniformly
+            in [1,3]. Defaults to -1.
+        alpha: a float value used as the probability coefficients for the
```

Comment:
are both distributions meant to be parameterized by the same alpha parameter?  Fine if so, just want to confirm.

Reply:
Yes, that's the way it has been done in the paper.